cmake/developer_package/IEDevScriptsConfig.cmake

```diff
@@ -4,7 +4,27 @@
 
 cmake_minimum_required(VERSION 3.13)
 
+if(NOT DEFINED IEDevScripts_DIR)
+    message(FATAL_ERROR "IEDevScripts_DIR is not defined")
+endif()
+
+set(OLD_CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH})
+list(APPEND CMAKE_MODULE_PATH "${IEDevScripts_DIR}")
+
+function(set_ci_build_number)
+    set(repo_root "${CMAKE_SOURCE_DIR}")
+    include(version)
+    set(CI_BUILD_NUMBER "${CI_BUILD_NUMBER}" PARENT_SCOPE)
+endfunction()
+
+set_ci_build_number()
+
+include(features)
```

Comment:
Maybe it will be safer to use full path here, like `include("${CMAKE_CURRENT_LIST_DIR}/features.cmake")`.

Just, if some 3rd-party project has its own `features.cmake` module and it was added to `CMAKE_MODULE_PATH` prior to `IEDevScripts` inclusion, this script might use 3rd-party project's file instead its own.

Reply:
Or at least instead of appending `CMAKE_MODULE_PATH` just overwrite it.

Reply:
fixed using last suggestion.